mne/preprocessing/nirs/_tddr.py

```diff
@@ -30,6 +30,11 @@ def temporal_derivative_distribution_repair(raw):
     raw : instance of Raw
          Data with TDDR applied.
 
+    Notes
+    -----
+    There is a shorter alias :func:`mne.preprocessing.nirs.tddr` that can be
```

Comment:
Use double-backticks here. There is no entry in `python_reference.rst` for this to link to, hence on CircleCI:
```
/home/circleci/project/mne/preprocessing/nirs/_tddr.py:docstring of mne.preprocessing.nirs.temporal_derivative_distribution_repair:30: WARNING: py:func reference target not found: mne.preprocessing.nirs.tddr
```

Reply:
Done